packages/valory/skills/safe_deployment_abci/__init__.py

```diff
@@ -19,7 +19,22 @@
 
 """This module contains the ABCI safe deployment skill for an AEA."""
 
+from enum import Enum
+
 from aea.configurations.base import PublicId
 
 
 PUBLIC_ID = PublicId.from_str("valory/safe_deployment_abci:0.1.0")
+
+
+class States(Enum):
+    """Enumeration of states."""
+
+    DEPLOY_SAFE = "deploy_safe"
+    VALIDATE_SAFE = "validate_safe"
+    RANDOMNESS_SAFE = "randomness_safe"
+    SELECT_KEEPER_SAFE = "select_keeper_safe"
+
+    def __str__(self) -> str:
+        """Get the string value of the state."""
+        return self.value
```

Comment:
Could be dropped for consistency

Reply:
this States enumeration looks unnecessary to me. We have rounds, behaviours and payloads, they map one to one. We do not need `TransactionType` or `States` I think, but I would need to see a design on paper, perhaps it does make sense in a scenario I did not envision.

Reply:
Somewhere one has to define the identifiers. It's better to do this in one place than in two